src/bin/inx-chronicle/cli.rs

```diff
@@ -1,30 +1,35 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use clap::Parser;
+use clap::{Parser, Subcommand};
+
+use crate::config::{ChronicleConfig, ConfigError};
 
 /// Chronicle permanode storage as an INX plugin
 #[derive(Parser, Debug)]
 #[clap(author, version, about)]
 pub struct ClArgs {
     /// The location of the configuration file.
-    #[clap(short, long)]
+    #[clap(short, long, env = "CONFIG_PATH")]
     pub config: Option<String>,
-    /// The address of the MongoDB database.
-    #[clap(long = "db")]
-    pub db_addr: Option<String>,
+    /// The MongoDB connection string.
+    #[clap(long, env = "MONGODB_CONN_URI")]
+    pub mongodb_conn_url: Option<String>,
```

Comment:
```suggestion
    pub mongodb_url: Option<String>,
```

Reply:
Any reasoning? I think it's good to specify that this isn't simply the address, but the connection string.

Reply:
I am fine with that. @grtlr ?

Reply:
I'm sorry I just now learned that there is more than just an URL to this. I'm fine with `mongodb_conn_str`.